packages/universal-xml/lib/node-map.js

```diff
@@ -0,0 +1,268 @@
+/*
+ * This file exports a mapping between universal node names and platform-specific node names. The
+ * keys in the exported object are the universal node names, and the values are objects with keys
+ * corresponding to platformNames. The values of those platformName keys can be either a string
+ * (meaning that that platform-specific node name will be mapped to the universal node name), or an
+ * array of strings. This means that there can be a many-to-one relationship between
+ * platform-specific node names and universal node names.
+ */
+export default {
```

Comment:
(haven't fully read yet)

Question:
So, when someone wants to add Flutter elements as `alert` or something, can they create a PR to append them here?
e.g.

A someone wants to add FlutterAlertElement in flutter app (not sure like this element exactly exists though), they can add it like below to handle the element as 'alert'

```
  Alert: {
    ios: ['XCUIElementTypeAlert', 'FlutterAlertElement'],
    android: ['android.widget.Toast', 'FlutterAlertElement'],
  },
```

Reply:
Yes, that's correct. Right now it is based on platform, but I suppose it could be based on automationName instead, so it could look like: 

```
Alert: {
  xcuitest: ['XCUIElementTypeAlert'],
  uiautomator2: ['android.widget.Toast'],
  espresso: ['android.widget.Toast'],
  flutter: ['FlutterAlertElement'],
}
```

either way  it seems we run into some duplication (some drivers have same elements across multiple platforms, whereas some platforms have same elements across multiple drivers)